src/app/components/ActiveUserLoanContainer/components/ActiveLoanTableMobile/index.tsx

```diff
@@ -24,33 +26,68 @@ interface Props {
     profit: any;
     startMargin: string;
     startPrice: string;
+    currency: any;
   }>;
   setExpandedId: any;
   expandedId: string;
 }
 
 export function ActiveLoanTableMobile(props: Props) {
   const rows = props.data.map(item => {
-    return item.id === props.expandedId ? (
-      <ExpandedRowMobile
-        key={item.id}
-        item={item}
-        handleClick={() => props.setExpandedId('')}
-      />
-    ) : (
-      <div
-        key={item.id}
-        className="row mobile-row"
-        onClick={() => {
-          props.setExpandedId(item.id);
-        }}
-        style={{ opacity: props.expandedId ? '0.2' : '1' }}
-      >
-        <div className="col-2">{item.icon}</div>
-        <div className="col-4">{item.positionSize}</div>
-        <div className="col-3">{item.profit}</div>
-        <div className="col-3">{item.mobileActions}</div>
-      </div>
+    const expanded = item.id === props.expandedId || !props.expandedId;
+    return (
+      <>
+        <div
+          key={item.id}
+          className={`row mobile-row ${!expanded && 'opaque'}`}
+          onClick={() => {
+            props.setExpandedId(props.expandedId === item.id ? '' : item.id);
+          }}
+        >
+          <div className="col-2">
+            {item.icon === 'LONG' && (
+              <Icon
+                icon="circle-arrow-up"
+                className="text-customTeal mx-2"
+                iconSize={20}
+              />
+            )}
+            {item.icon === 'SHORT' && (
+              <Icon
+                icon="circle-arrow-down"
+                className="text-Gold ml-2"
+                iconSize={20}
+              />
+            )}
+          </div>
+          <div className="col-4">
+            {formatAsBTC(item.positionSize, item.currency)}
+          </div>
+          <div
+            className={`col-3 ${item.profit > 0 ? 'text-green' : 'text-red'}`}
+          >
+            ${' '}
+            {item.profit.toLocaleString('en', {
+              maximumFractionDigits: 4,
+              minimumFractionDigits: 4,
+            })}
```

Comment:
For currency it's better to use toLocaleString with currency type:
```
{item.profit.toLocaleString('en', {
    style: 'currency',
    currency: 'USD',
    maximumFractionDigits: 4,
    minimumFractionDigits: 4,
})}
```
Instead of specifying locale `'en'` we can also set it to `undefined` - app would use users preferred locale in this case.

Also, this code is repeated for couple of times - maybe it's a good chance to create new helper function.

Reply:
I've added in some helper functions, thanks! For currencies, I set the locale as 'en-US', because otherwise the price looks like 'US$1000' instead of '$1000' which I thought looked a bit crowded.

Reply:
Thanks 💯